cmd/claimserver/endpoint/admin.go

```diff
@@ -0,0 +1,66 @@
+package endpoint
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+)
+
+func handleInfo(c *gin.Context) {
+	r := adminservice.Info()
+	c.JSON(200, gin.H{
+		"info": r,
+	})
+}
+
+func handleRawDump(c *gin.Context) {
+	r := adminservice.RawDump()
+	c.JSON(http.StatusOK, r)
+}
+
+func handleRawImport(c *gin.Context) {
+	var data map[string]string
+	c.BindJSON(&data)
+
+	count, err := adminservice.RawImport(data)
+	if err != nil {
+		c.String(http.StatusBadRequest, err.Error())
+	}
+	c.String(http.StatusOK, "imported "+strconv.Itoa(count)+" key,value entries")
+}
+
+func handleClaimsDump(c *gin.Context) {
+	r := adminservice.ClaimsDump()
+	c.JSON(http.StatusOK, r)
+}
+
+type addClaimBasicMsg struct {
```

Comment:
This could look nicer with a json.Unmarshaler that also validates the input.  Let's talk about this later.

Reply:
agree, after this we can make a branch and abstract those parts of code for all the servers

Reply:
We leave this for future iterations.